src/classes/User.js

```diff
@@ -0,0 +1,41 @@
+class User {
+  constructor(userData = {}) {
+    this.id = userData.id;
+    this.name = userData.name;
+    this.pantry = userData.pantry;
+    this.recipesToCook = [];
+  }
+
+  addRecipesToCook(recipe) {
+    this.recipesToCook.push(recipe);
+  }
+  
+  removeRecipesToCook(recipe) {
+    let recipeToRemove = this.recipesToCook.indexOf(recipe);
+    this.recipesToCook.splice(recipeToRemove, 1);
+    return this.recipesToCook;
+  }
+  filterRecipeByTag(tag) {
+    let filteredRecipesByTag = this.recipesToCook.filter((recipe) => {
+      if (recipe.tags.includes(tag)) {
+        return true;
+      }
+    });
+    return filteredRecipesByTag;
+  }
+  userFilteredRecipesByName(recipeName) {
+    let userFilteredRecipe = this.recipesToCook.filter((recipe) => {
```

Comment:
would prefer plural `recipes` for this variable name since its value would be an array of recipes rather than just one recipe. `recipesFilteredByName` would be a good option

Reply:
change to `filteredRecipesByName` to match the filter by recipes tag method